.goreleaser.yml

```diff
@@ -0,0 +1,58 @@
+archives:
+  - builds:
+      - jb
+    format: tar.gz
+    format_overrides:
+      - format: zip
+        goos: windows
+    name_template: '{{ .ProjectName }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}-{{ .Mips }}{{ end }}'
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+brews:
+  - description: The jsonnet-bundler is a package manager for Jsonnet.
+    homepage: https://github.com/jsonnet-bundler/jsonnet-bundler
+    skip_upload: true
+    tap:
+      name: jsonnet-bundler
+      owner: jsonnet-bundler
+builds:
+  - binary: jb
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 6
+      - 7
+    goos:
+      - darwin
+      - linux
+      - windows
```

Comment:
Will that also attempt windows/arm builds, etc? I think we only need:

```
linux/amd64
linux/arm64
linux/arm
darwin/amd64
windows/amd64
```

and due to apple silicon maybe soon `darwin/arm64`

Reply:
For now, it's only generating the ones you mentioned. If you look at the `tree` output above in the PR comments, it generates the following archives:

```
jb-darwin-amd64.tar.gz
jb-linux-amd64.tar.gz
jb-linux-arm64.tar.gz
jb-linux-armv6.tar.gz
jb-linux-armv7.tar.gz
jb-windows-amd64.zip
```

Reply:
Pushed just now.